feature/main/src/main/java/com/readme/android/main/viewmodel/MainViewModel.kt

```diff
@@ -13,6 +14,10 @@ class MainViewModel @Inject constructor() : ViewModel() {
     private val _isMyFeed = MutableEventFlow<Boolean>()
     val isMyFeed
         get() = _isMyFeed.asEventFlow()
+    var isCategorySelected = MutableLiveData(true)
+        private set
+    var selectedCategory = MutableLiveData("에세이, 경제/경영 외 2개") // 수정예정
+        private set
```

Comment:
이건 단순히 궁금한 것인데 그냥 이렇게 정의하는 거면
```suggestion
    val isCategorySelected = MutableLiveData(true)
    val selectedCategory = MutableLiveData("에세이, 경제/경영 외 2개") // 수정예정
```
1. 이거랑 별반 다름없이 작동할 것 같은데 우선 `var`키워드로 정의한 이유가 있는지 궁금하고
2. ViewModel과 나중에 양방향 데이터 바인딩을 쓰려고 `private`접근자를 달지 않은 건지 궁금합니당..! 그렇지않다면 캡슐화를 위해 private 접근자를 달아주는 게 좋을것같아요..!

Reply:
ㅋㅋㅋㅋ이제야 리뷰 반영하러 왔어유..
1. val은 제가 실수한거네유!
2. 양방향 데이터바인딩은 아니구, `HomeFragment.kt` 에서 이렇게 가져다 씁니다!

```kotlin
  val homeHeaderAdapter =
      viewModel.isCategorySelected.value?.let {
          HomeHeaderAdapter(it, viewModel.selectedCategory.value, ::onCategoryIconClick)
      }
```

혹시 `getIsCategorySelected()` 같은 메소드를 생성하고, 해당 변수는 private으로 바꾸는게 좋을까요??